CenterlineMetrics/CenterlineMetrics.py

```diff
@@ -75,11 +75,6 @@ def setup(self):
     self.ui.inputModelSelector.connect("currentNodeChanged(vtkMRMLNode*)", self.onSelectNode)
     self.ui.outputPlotSeriesSelector.connect("currentNodeChanged(vtkMRMLNode*)", self.onSelectNode)
     self.ui.outputTableSelector.connect("currentNodeChanged(vtkMRMLNode*)", self.onSelectNode)
-    self.ui.radioR.connect("clicked()", self.onRadioR)
```

Comment:
I always felt that this was just a workaround and should not be necessary. It is great that this will be gone.